slither/printers/summary/cheatcodes.py

```diff
@@ -0,0 +1,73 @@
+"""
+    Cheatcode printer
+
+    This printer prints the usage of cheatcode in the code.
+"""
+from slither.printers.abstract_printer import AbstractPrinter
+from slither.slithir.operations import HighLevelCall
+from slither.utils import output
+
+
+class CheatcodePrinter(AbstractPrinter):
+
+    ARGUMENT = "cheatcode"
+
+    HELP = """
+        Print the usage of (Foundry) cheatcodes in the code.
+        For the complete list of Cheatcodes, see https://book.getfoundry.sh/cheatcodes/
+    """
+
+    WIKI = "https://github.com/trailofbits/slither/wiki/Printer-documentation#cheatcode"
+
+    def output(self, filename: str) -> output.Output:
+
+        info: str = ""
+
+        try:
+            vm = self.slither.get_contract_from_name("Vm").pop()
+        except IndexError:
+            return output.Output("No contract named VM found")
+
+        for contract in self.slither.contracts_derived:
+            # Check that the IS_TEST variable is set. (Only works for Foundry)
+            is_test_var = contract.variables_as_dict.get("IS_TEST", None)
+            is_test = False
+            if is_test_var is not None:
+                try:
+                    is_test = is_test_var.expression.value == "true"
+                except AttributeError:
+                    pass
+
+            if not is_test:
+                continue
```

Comment:
Do we also want to flag cheatcode issue in non test files?

Reply:
I don't think so